heartbeat/mysql-common.sh

```diff
@@ -239,8 +245,8 @@ mysql_common_start()
     --datadir=$OCF_RESKEY_datadir \
     --log-error=$OCF_RESKEY_log \
     $OCF_RESKEY_additional_parameters \
-    $mysql_extra_params >/dev/null 2>&1" &
-    pid=$!
+    $mysql_extra_params >/dev/null 2>&1 &
+    pid=$!"
```

Comment:
You shouldnt change this:
https://github.com/ClusterLabs/resource-agents/pull/1422/files

Reply:
Done.